Lib/test/test_subprocess.py

```diff
@@ -1661,6 +1662,78 @@ def test_start_new_session(self):
             child_pgid = int(output)
             self.assertNotEqual(parent_pgid, child_pgid)
 
+    def test_user(self):
+        # For code coverage of the user parameter.  We don't care if we get an
+        # EPERM error from it depending on the test execution environment, that
+        # still indicates that it was called.
+
+        uid = os.getuid()
+        for user in [65534 if uid != 65534 else 65533,
+                     "nobody" if sys.platform != 'darwin' else "unknown"]:
+            try:
+                output = subprocess.check_output(
```

Comment:
Looks like a good place for `unittest.TestCase.subTest`. Same with the other test.

Reply:
OK, will do